### PR TITLE
Friendly web interface

### PR DESCRIPTION
**Pull request by Kerollmops**
Here is new web interface, based on Bulma + some ugly hacks to show the documents.

Screenshot_2019-12-14 MeiliSearch

### CODE REVIEW

**Comment by Kerollmops** [comment]
I do not like the search panel, where the numbers are really far on the right and the search bar far on the left.

File: meilisearch-http/public/interface.html [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/bulma.min.css">
+    <title>MeiliSearch</title>
+    <style>
+      em {
+        color: hsl(204, 86%, 25%);
+        font-style: inherit;
+        background-color: hsl(204, 86%, 88%);
+      }
+
```
Comment (qdequele) [inline meilisearch-http/public/interface.html]: Add this code to center the search bar.

```css
.notification {
    display: flex;
    justify-content: center;
}
.level-left{
    margin-right: 50px;
}
```

Comment (Kerollmops) [reply meilisearch-http/public/interface.html]: Thank you!

**Comment by bidoubiwa** [comment]
documentation : meilisearch/documentation#112

**Comment by anborg** [comment]
The frontend is great.  

Adding a small vuejs/angular snippet for search box, and result list, would increase adoption  - as t is easy to configure such js,change  hostName and indexName.

Bundling a primitive crawl script that could index a given url into "myinternalwebsiteIndex" would help for internal websites.

**Comment by Kerollmops** [comment]
Hey @anborg, We are currently working on making a [javascript library first](https://github.com/meilisearch/searchbar.js). Once done we will work on making React and VueJs libraries from that one.

**Pull request merged**